### DISCUSSION TITLE
Run directus in development mode

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#13439] [Archive]
Hi all!

I've added directus as a dependency to my project. When developing I'm running this dev script from my package.json:
```
"dev": "yarn install; npx nodemon --legacy-watch --exec NODE_OPTIONS=--inspect=0.0.0.0:9229 DEV=true MODE=development NODE_ENV=development directus start",
```

I've just started working on some app extensions, however, no matter what environment variable I set to development, the console of the app always shows the environment to be production. Therefor the VueTools extension doesn't work.

How can I override the mode of the app and set it to development?

Cheers!

Kees.

**Comment by br41nslug** [comment]
To run directus in development mode you'll need to run it from source instead of from the npm package.
https://docs.directus.io/contributing/running-locally/

**Reply by keesvanbemmel** [reply]
But where do I put my custom module extensions in that case? They won't load from src?

**Reply by nickrum** [reply]
They should be loaded from `/api/extensions/modules/` (provided you didn't change the `EXTENSIONS_PATH` env var).
